desc/objectives/_coils.py

```diff
@@ -0,0 +1,472 @@
+"""Objectives for targeting geometrical quantities."""
+
+import warnings
+
+import numpy as np
+
+from desc.backend import jnp
+from desc.compute import compute as compute_fun
+from desc.compute import get_profiles, get_transforms, rpz2xyz, rpz2xyz_vec
+from desc.compute.utils import safenorm
+from desc.grid import LinearGrid
+from desc.utils import Timer, warnif
+
+from .objective_funs import _Objective
+
+
+class QuadraticFlux(_Objective):
```

Comment:
Should combine with #722

Reply:
Kian's is pointing into free bdry currently, this is going to master. I will help combine once this is in master and Kian's points to master again